### PR TITLE
refactor: encapsulate codes into function FormatStatus

### PR DESCRIPTION
**Pull request by zeppp**
Signed-off-by: zeppp <zeppp1995@gmail.com>

**1.Describe what this PR did**
encapsulate codes into function FormatStatus
**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by allencloud** [comment]
Thanks a lot for your work. 🌹 
Please add a unit test for this function. @zeppp

File: daemon/mgr/container_types.go [hunk 1]
```diff
@@ -154,6 +156,27 @@ func (meta *ContainerMeta) merge(getconfig func() (v1.ImageConfig, error)) error
 	return nil
 }
 
+// FormatStatus format container status
+func (meta *ContainerMeta) FormatStatus() (string, error) {
+	var status string
+
+	if meta.State.Status == types.StatusRunning || meta.State.Status == types.StatusPaused {
+		start, _ := time.Parse(utils.TimeLayout, meta.State.StartedAt)
```
Comment (allencloud) [inline daemon/mgr/container_types.go]: Please do not ignore this error.

**Comment by zeppp** [comment]
Should I add unit test in utils_test.go or somewhere?  @allencloud

**Comment by allencloud** [comment]
In `daemon/mgr/container_types_test.go`, so you can add this file to the current directory.

File: apis/server/container_bridge.go [hunk 2]
```diff
@@ -244,22 +244,13 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
...
-				return err
-			}
-
-			container.Status = "Up " + startAt
-			if m.State.Status == types.StatusPaused {
-				container.Status += "(paused)"
-			}
-		} else {
-			container.Status = string(m.State.Status)
+		status, err := m.FormatStatus()
```
Comment (allencloud) [inline apis/server/container_bridge.go]: Move this part above above `container := types.Container{` and still in the for loop, then we can add status when constructing the instance, like:
```
		container := types.Container{
			ID:         m.ID,
			Names:      []string{m.Name},
			Image:      m.Config.Image,
			Command:    strings.Join(m.Config.Cmd, " "),
			Created:    t.UnixNano(),
			Labels:     m.Config.Labels,
			HostConfig: m.HostConfig,
            Status: status,
		}
```

**Comment by allencloud** [comment]
LGTM, much better now.

**Pull request merged**